rotkehlchen/chain/scroll/decoding/decoder.py

```diff
@@ -42,6 +43,7 @@ def __init__(
                 is_non_conformant_erc721_fn=self._is_non_conformant_erc721,
                 address_is_exchange_fn=self._address_is_exchange,
             ),
+            dbevmtx_class=DBOptimismTx,
```

Comment:
Is this needed? It's already the default argument no?

Also shouldnt' it be rename to L1WithL2FeesTx?